### DISCUSSION TITLE
App starter request // with bundler, routing, protected routes, and theme

### DISCUSSION THREAD

**Discussion by sagaripte** [luwes/sinuous#169] [Ideas]
Hi... May I please request an example/starter template, with various dependencies required to create an app.

Ideally should have bundler (maybe poi.js.org), public routes, protected routes, and a good theme (maybe Bulma)

would be even better if someone can showcase a real-time example with socket.io or similar library

Sorry I am new to these technologies and there are so many options available that its confusing to figure out best practices.

**Comment by luwes** [comment]
@sagaripte thank you for the contribution, I really appreciate it!

At the moment I have little time to put a starter template together but there are some really good examples made already that could be of some use.

https://codesandbox.io/s/sinuous-router-g2eud
https://codesandbox.io/s/sinuous-hacker-news-dqtf7
https://codesandbox.io/s/github/theSherwood/7_GUIs/tree/master/sinuous
https://github.com/johannschopplich/plain-spa
